templates/stripes/stripe-toc.php

```diff
@@ -0,0 +1,31 @@
+<?php $toc = $this->tableOfContents($scope) ?>
+
+<?php if (!empty($toc)) : ?>
+<div class="<?= $this->e($type, 'wrapperClasses') ?>">
+  <div class="vssl-stripe-column">
+    <div class="vssl-stripe--card">
+      <?php if (!empty($title['html'])) : ?>
+        <div class="vssl-stripe--toc--title"><?= $this->inline($title['html']) ?></div>
```

Comment:
Are you thinking that the API would return the default title here if the user hasn't added one? Or do we need to be checking for the default here if `$title['html']` is empty?

Reply:
I made it so the frontend editor code will automatically put "Table of Contents" as the default title. But I can add a check here too to be sure / futureproof.